backend/tests/conftest.py

```diff
@@ -946,6 +948,11 @@ def mock_epoch_details(mocker, graphql_client):
     mock_graphql(mocker, epochs_events=list(EPOCH_EVENTS.values()))
 
 
+@pytest.fixture(scope="function")
+def mock_gql_transport_reorg_error(mocker):
```

Comment:
is this wrapper needed for mock_failing_gql or we can just use mock_failing_gql directly?

Reply:
+1